lib/src/parameters.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:convert';
+import 'package:collection/collection.dart';
+import 'package:http_parser/http_parser.dart';
+
+typedef Map<String, dynamic> GetParameters(String contentType, String body);
+
+void validate(bool condition, String message) {
```

Comment:
I don't think it's worth defining this versus just manually writing `if` statements in `parseJsonParameters()`.